netsync/manager.go

```diff
@@ -727,6 +734,10 @@ func (sm *SyncManager) handleInvMsg(imsg *invMsg) {
 		}
 
 		if iv.IsBlockOrSyncBlock() {
+			if !peer.Inbound() && sm.dag.IsKnownInvalid(iv.Hash) {
```

Comment:
Why is peer.Inbound() has to do with anything?
If there's a good reason please add comment.